src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -1165,6 +1165,24 @@ CrontabSchedule VerifyBackupFrequency(string backupFrequency)
             }
         }
 
+        [RavenAction("/admin/unused-databases", "POST", AuthorizationStatus.Operator)]
```

Comment:
Can we change that to `/admin/databases/unused-ids`? Will match other endpoints here

Reply:
done